spec/system/topic_list/glimmer_spec.rb

```diff
@@ -73,4 +70,33 @@
       ).to eq("3")
     end
   end
+
+  describe "topic highlighting" do
+    # TODO: Those require `Capybara.disable_animation = false`
```

Comment:
See also: 

https://github.com/discourse/discourse/blob/main/spec/system/composer/dont_feed_the_trolls_popup_spec.rb#L15C1-L18C1

@tgxworld, if you want to restart the work on it - I'm up for inter-team collaboration 🤪